tests/test_custom_environments.py

```diff
@@ -0,0 +1,53 @@
+from pilco.environments import Environment
+from pilco.errors import EnvironmentError
+
+import numpy as np
+
+
+def test_custom_environment_runs():
```

Comment:
There is currently no assertion in this test. If you initialize these to a stationary position, then it would be nice to assert that at the end the state is still (close to) the initial one.

Reply:
With this test I intended to check if the environment will run with no errors. That's a fair point, we could probably check that the initial and final states are close.

Reply:
Actually let's maybe leave this for another iteration? I think we should define the API for environments first, which could also define the stationary states for each environment. That way we can do this check (whether the system stays in a stationary state when initialised there) automatically instead of having to hard code the initialisation for this check.